hw_diag/templates/diagnostics_page.html

```diff
@@ -46,7 +46,9 @@ <h3 class="text-center mb-4">Diagnostics Breakdown</h3>
                 </tr>
                 <tr>
                   <td>Sync Percentage</td>
-                  {% if diagnostics.BSP > 99.98 %}
+                  {% if not diagnostics.BSP %}
+                    <td class="text-right">API Not Available</td>
```

Comment:
"Helium API'

Reply:
> "Helium API'

I changed it to say Helium API Not Available, will push soon